ez/6.x/Dockerfile.tmpl

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/continuouspipe/symfony-php{{PHP_VERSION}}-apache:latest
```

Comment:
how much of Symfony does it use? If not supporting the symfony bundle system I think we should do as I did with Spryker. All the symfony_functions.sh assume there's the symfony console tool